tests/python/gaia-ui-tests/gaiatest/apps/lockscreen/app.py

```diff
@@ -110,6 +111,9 @@ def a11y_click_camera_button(self):
 
     @property
     def notifications(self):
+        container = self.marionette.find_element(*self._lockscreen_container_locator)
+        if not container.is_displayed() or container.text == '':
+            return []
```

Comment:
So this is to avoid the search timeout when there are no notifications? It seems reasonable enough to me, but would be good to include a comment.